client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html

```diff
@@ -0,0 +1,51 @@
+<div id="openstackClient" data-ng-controller="CloudProjectComputeInfrastructureOpenstackClientCtrl as OpenstackClientCtl" >
+    <div class="margin"></div>
+    <div class="terminal col-md-offset-2 col-sm-offset-3"
+     data-ng-class="{ maximized : OpenstackClientCtl.maximized, minimized : OpenstackClientCtl.minimized }"
+     >
+        <div class="header" data-ng-click="OpenstackClientCtl.clickBar()">
+            <div class="title" data-translate="cpci_openstack_client">
+            </div>
+            <div class="buttons">
+                <button class="no-style" type="button"
+                 data-ng-if="!OpenstackClientCtl.minimized"
+                 data-ng-click="OpenstackClientCtl.minimize();$event.stopPropagation()"><i class="fa fa-window-minimize"></i></button>
+                <button class="no-style" type="button" data-ng-click="OpenstackClientCtl.maximize();$event.stopPropagation()"><i class="fa fa-window-maximize"></i></button>
+            </div>
+        </div>
+        <div class="settings">
+            <label data-translate="cpci_openstack_client_regions"></label>
+            <select ng-if="!OpenstackClientCtl.regions.loading"
+                class="form-control"
+                ng-change="OpenstackClientCtl.Service.setRegion(OpenstackClientCtl.region)"
+                ng-model="OpenstackClientCtl.region"
+                >
+                    <option value="{{OpenstackClientCtl.emptyOption}}" hidden data-translate="cpci_openstack_client_select_region"></option>
+                    <option ng-repeat="region in OpenstackClientCtl.regions.data track by region" value="{{region}}">
```

Comment:
select ng-options should cover that for you... but I remember you had issue with this T_T

Reply:
yes I find out the bug but I need the first hidden option